.github/workflows/plugins-test.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Run okhttp 3.0.x-3.14.x (34)
         run: bash test/plugin/run.sh okhttp-scenario
 
-  Spring31x_ES5_Gateway_Avro:
+  Spring31x_ES5_Gateway_Avro_Struts2.5:
```

Comment:
`.` is not valid as a yaml section name, unless it’s quoted

Reply:
done